apps/gro/controllers/address-start.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const BaseController = require('hof').controllers.base;
+const Client = require('node-rest-client').Client;
+const config = require('../../../config');
+var logger = require('../../../lib/logger');
+
+module.exports = class AddressStartController extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  getValues(req, res) {
+    const postcode = req.sessionModel.attributes.postcode.toUpperCase();
+    req.sessionModel.set('postcode', postcode);
```

Comment:
also we should use the correct model api - not set/get values using `attributes` - use `.get` and `.set`